jsapp/js/components/activity/formActivity.tsx

```diff
@@ -1,7 +1,116 @@
+import type {ReactNode} from 'react';
+import {useState} from 'react';
+import '../../../scss/components/_kobo.form-view.scss';
+import type {KoboSelectOption} from '../common/koboSelect';
+import KoboSelect from '../common/koboSelect';
+import './formActivity.scss';
+import type {UniversalTableColumn} from 'jsapp/js/universalTable/universalTable.component';
+import Button from '../common/button';
+import PaginatedQueryUniversalTable from 'jsapp/js/universalTable/paginatedQueryUniversalTable.component';
+import type {ActivityLogItem} from 'jsapp/js/query/queries/activityLog.query';
+import {useActivityLogQuery} from 'jsapp/js/query/queries/activityLog.query';
+import moment from 'moment';
+
+interface TableDataItem {
+  description: ReactNode;
+  date: ReactNode;
+}
+
+const mockOptions: KoboSelectOption[] = [
+  {value: '1', label: 'Option 1'},
+  {value: '2', label: 'Option 2'},
+  {value: '3', label: 'Option 3'},
+];
+
+const columns: UniversalTableColumn[] = [
+  {
+    key: 'description',
+    label: t('Event description'),
+  },
+  {
+    key: 'date',
+    label: t('Date'),
+    size: 100,
+  },
+];
+
+const EventDescription = ({
+  who,
+  action,
+  what,
+}: {
+  who: string;
+  action: string;
+  what: string;
+}) => (
+  <div className='event-description'>
+    <div className='profile-picture'>{who.slice(0, 1)}</div>
+    <span className='description'>
+      <span className='who'>{who}</span>{' '}
+      <span className='action'>{action}</span> {what}
+    </span>
+    <button className='see-details'>{t('See details')}</button>
```

Comment:
Just to be sure it's clear (for further PRs) - we are not doing the "expanded row" kind of details. We are going with a modal solution :)